### DISCUSSION TITLE
RTC

### DISCUSSION THREAD

**Discussion by gbartling** [wills106/homeassistant-solax-modbus#395] [General]
Hi, i noticed today that my rtc is being updated every 15 seconds, not sure if it has always done this or since the update 2 days ago, is this normal? i've done a search for this issue but not found anything. i'm using the solis plugin and have been very happy with it so far so a big thank you to wills106!

Gary

**Comment by wills106** [comment]
The RTC updates at the same speed as you have set the polling frequency to.

So I am guessing you have your polling frequency set at 15s?

How I imagined RTC to be used is.. To turn it on every so often to check if it's drifted out.
Then based on that set an automation once per month for example to press the RTC Sync button to bring it back in sync.
So the majority of the time RTC is disabled.

**Reply by gbartling** [reply]
thanks wills106 for the prompt reply. i too thought it would only need to update once every month so was surprised to see it happening every 15 seconds but if this is normal i will re enable the integration and carry on using this great program. thanks again.

Gary
